### ISSUE TITLE
Game view for simple shapes in URDF importer

### ISSUE THREAD

**Issue by michalpelka** [o3de/o3de-extras#147]
**Is your feature request related to a problem? Please describe.**
The created prefab has simple shapes (like box, cylinder, sphere) created as e.g. `EditorSphereShapeComponent`.
The created has option "Game view" disabled (see attached screenshot), so it is not visible after hitting CRTL+G.
![image](https://user-images.githubusercontent.com/3209244/189311288-e55309d3-6a8f-48b6-bfbd-66c324c095b5.png)

**Describe the solution you'd like**
I would like to have properly drawn simple shapes in the game mode. Ideally without opacity.

**Additional context**
 **Minimal URDF**:
 ```
 <robot name="test">
    <link name="base">
        <inertial>
        <origin xyz="0. 0. 0."/>
        <mass value="1."/>
        <inertia ixx="1." ixy="0." ixz="0." iyy="1." iyz="0." izz="1."/>
        </inertial>
        <visual>
            <origin rpy="0 0 0" xyz="0 0 0"/>
            <geometry>
                <box size="1 1 1"/>
            </geometry>
        </visual>
        <collision>
            <origin rpy="0 0 0" xyz="0 0 0"/>
            <geometry>
                <box size="1 1 1"/>
            </geometry>
        </collision>
    </link>
</robot>
```

**Comment by michalpelka** [comment]
It is still occurring. Fixed in https://github.com/o3de/o3de-extras/pull/169

**Comment by michalpelka** [comment]
Closed by merging #169.
